dev/devicelab/lib/framework/devices.dart

```diff
@@ -364,9 +365,8 @@ class FuchsiaDeviceDiscovery implements DeviceDiscovery {
   /// Picks the first connected Fuchsia device.
   @override
   Future<void> chooseWorkingDevice() async {
-    final List<FuchsiaDevice> allDevices = (await discoverDevices())
-      .map<FuchsiaDevice>((String id) => FuchsiaDevice(deviceId: id))
-      .toList();
+    final List<FuchsiaDevice> allDevices =
+        (await discoverDevices()).map<FuchsiaDevice>((String id) => FuchsiaDevice(deviceId: id)).toList();
```

Comment:
This line doesn't match the style guide's advice about keeping things on the same line. Previous formatting was more readable IMHO

Reply:
Reverted back to previous formatting to simplify the code review